picoquictest/stresstest.c

```diff
@@ -825,7 +825,7 @@ int stress_test()
 #else
             "certs/cert.pem", "certs/key.pem",
 #endif
-            PICOQUIC_TEST_ALPN, stress_server_callback, (void*)&stress_ctx, NULL, NULL, NULL,
+            PICOQUIC_TEST_ALPN, stress_server_callback, NULL, NULL, NULL, NULL,
```

Comment:
Wow. Yes, you are right, it does not belong there. I wonder why the test was actually working on Windows...